src/daft-local-execution/src/lib.rs

```diff
@@ -14,16 +14,21 @@ lazy_static! {
     pub static ref NUM_CPUS: usize = std::thread::available_parallelism().unwrap().get();
 }
 
+pub(crate) type TaskSet<T> = tokio::task::JoinSet<T>;
+pub(crate) fn create_task_set<T>() -> TaskSet<T> {
+    tokio::task::JoinSet::new()
+}
```

Comment:
Why wrap `tokio::task::JoinSet` here?

Reply:
I wanna keep the operators agnostic from the implementation details, to reduce complexity.